dgraph/cmd/zero/http.go

```diff
@@ -231,9 +234,46 @@ func (st *state) getState(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) zeroHealth(ctx context.Context) (*api.Response, error) {
+	if ctx.Err() != nil {
+		return nil, ctx.Err()
```

Comment:
Can you wrap this error also? It would help to figure out where the error is actually coming from